backend/src/core/models/games.aggregate.js

```diff
@@ -34,4 +34,20 @@ export class GamesAggregate {
   #findPageForGameById(htmlDetailsPages, gameId) {
     return htmlDetailsPages.find((page) => gameId === page.id).page;
   }
+
+  extractReleaseDatesViaSteamApi(steamApiApps) {
+    this.#games = this.#games.map((game) => {
+      const gameCopy = game.copy();
+
+      const steamApiApp = this.#findGameByAppId(steamApiApps, gameCopy.id);
+
+      gameCopy.updateReleaseDateViaSteamApi(steamApiApp);
+
+      return gameCopy;
+    });
+  }
+
+  #findGameByAppId(steamApiApps, gameId) {
+    return steamApiApps.find((app) => app.steam_appid === gameId);
```

Comment:
Change according to: https://github.com/lukatarman/steam-game-stats/pull/222#discussion_r1604884159